Resources/Prototypes/_NF/Hydroponics/seeds.yml

```
@@ -173,3 +173,32 @@
       Min: 1
       Max: 10
       PotencyDivisor: 10
+
+- type: seed
+  id: monkanana
+  name: seeds-monkanana-name
+  noun: seeds-noun-seeds
+  displayName: seeds-monkanana-display-name
+  plantRsi: _NF/Objects/Specific/Hydroponics/monkanana.rsi
+  packetPrototype: MonkananaSeeds
+  productPrototypes:
+    - FoodMonkana
+  harvestRepeat: Repeat
+  lifespan: 50
+  maturation: 6
+  production: 6
+  growthStages: 4
+  potency: 10
+  yield: 3
+  idealLight: 5
```

Shouldn't this be consistent with bananas at 9?  Surely it's a tropical fruit.
```suggestion
  idealLight: 9
```